### ISSUE TITLE
Show more options for //DEPS assist

### ISSUE THREAD

**Issue by quintesse** [jbangdev/jbang-idea#55]
I'm not sure what the current filters are for the //DEPS assist, but if you have a line like:

```
//DEPS org.slf4j:slf4j-simple:
```

And hit Alt+Space at the end of that line it only seems to show the very latest version. I can understand that showing all versions might result in very long lists for some artifacts, but perhaps we can show the last 10-20 ones at least?

**Comment by linux-china** [comment]
Yes, the result list is from  ` fun suggestPrefix(groupId: String, artifactId: String,parameters: SearchParameters, consumer: Consumer<RepositoryArtifactData>)` in  DependencySearchService, and I will take a look.

**Comment by maxandersen** [comment]
The rewrite in PR #159 now returns up to 50 versions from Maven Central (newest-first) plus local Maven repository matches. Please try 0.100.0 and reopen if you'd like even more results.
